#Use EB provided libz 
#Januray 11, 2016 by B. Hajgato (Vrije Universiteit Brussel)
--- trinityrnaseq-2.1.1/trinity-plugins/Makefile.org	2015-10-15 14:07:11.000000000 +0200
+++ trinityrnaseq-2.1.1/trinity-plugins/Makefile	2016-01-11 11:07:47.399905959 +0100
@@ -18,7 +18,8 @@
 
 samtools:
 	tar xvf samtools-0.1.19.tar.bz2
-	cd samtools-0.1.19 && $(MAKE) LIBPATH=-ltinfo
+	sed -e 's/ -lz/ -L\$$\{EBROOTZLIB\}\/lib -lz/' -i samtools-0.1.19/misc/Makefile
+	cd samtools-0.1.19 && $(MAKE) LIBPATH=-L${EBROOTZLIB}/lib 
 	mv samtools-0.1.19/samtools ./BIN/.
 
 jellyfish:
